boa/build.py

```diff
@@ -100,6 +100,8 @@
     shell_path = "/bin/sh"
 elif utils.on_win:
     shell_path = "bash"
+elif "darwin" in sys.platform:
+    shell_path = "/bin/zsh"
```

Comment:
how about using `/bin/bash` here? it seems to be still available on OS X?

Reply:
This is definitely needed, otherwise compctl is not found.

Reply:
yeah that was an issue with our way of detecting shells in catkin's setup.sh :) luckily!